apidoc/Titanium/UI/Picker.yml

```diff
@@ -39,11 +39,10 @@ description: |
 extends: Titanium.UI.View
 excludes:
     properties: [
-        anchorPoint,backgroundDisabledColor,backgroundDisabledImage,backgroundFocusedColor,
+        anchorPoint,backgroundDisabledColor,backgroundDisabledImage,backgroundFocusedColor,children
         backgroundFocusedImage,backgroundGradient,backgroundImage,backgroundLeftCap,backgroundRepeat,
         backgroundSelectedColor,backgroundSelectedImage,backgroundTopCap,center,enabled,layout,tintColor,zIndex]
-    methods: [ remove, removeAllChildren ]
-
+    methods: [add, remove, removeAllChildren, replaceAt]
```

Comment:
According to the docs themselves, you add data to the picker using the `add` method... don't think it should be removed

Reply:
Good catch, updated!